lib/msal-react-native-android-poc/MSALModule/android/src/main/java/com/reactlibrary/MSALModule.java

```diff
@@ -159,7 +175,48 @@ public void onError(@NonNull MsalException exception) {
         });
     }
 
-    /*
+    /**
+     * acquireTokenSilent: attempts to obtain a token silently from the cache
+     * Parameters: string scopesValue (string containing scopes separated by a " "), Promise promise (returned to an async function)
+     */
+    @ReactMethod
+    public void acquireTokenSilent(String scopesValue, Promise promise) {
+        //get account
+        IAccount account = loadAccount();
+        if (account == null) {
```

Comment:
I think you can just do `if  (!account)` here.

Reply:
+1

Reply:
I'm getting a "bad operand type IAccount for unary operator '!'" error with (!account) ....

Reply:
@sangonzal